blog/content/post/2024-11-04-qubes-os-summit-post.md

```diff
@@ -0,0 +1,226 @@
+---
+title: 'Qubes Os Summit 2024'
+abstract: 'The sixth edition of the Qubes OS Summit, organized by Invisible
+Things Lab and 3mdeb, took place at the end of September'
+cover: /covers/image-file.png
+author: marta.witkowska, piotr.król
+layout: post
+published: false
+date: 2024-11-04
+archives: "2024"
+
+tags:
+  - QubesOS
+  - conference
+categories:
+  - Miscellaneous
+  - OS Dev
+
+---
+
+## Qubes OS Summit 2024
+
+The sixth edition of the Qubes OS Summit, organized by Invisible Things Lab and
+3mdeb, took place at the end of September. This year's event, held from
+September 20th to 22nd, was hosted in Berlin. It was not only an in-person
+gathering but also streamed live. The previous year’s conference was also hosted
+in Berlin and included a live online streaming for remote participants from
+[the first](https://www.youtube.com/watch?v=_UxndcxIngw&t=2s) and
+[the second](https://www.youtube.com/watch?v=xo2BVTn7ohs&t=3s) day of the
+summit.
+
+The summit was highly productive, facilitating the exchange of significant
+knowledge and expertise among participants. Contributions and insights shared
+during the event were greatly appreciated, as they advanced discussions on
+security, privacy, and open-source technologies.
+
+## Day 1
+
+The first day of the Qubes OS Summit 2024 began with a
+[Welcome to Qubes OS Summit 2024](https://www.youtube.com/watch?v=lJFxtdan9qY)
+presentation held by Piotr Król and Marek Marczykowski-Górecki. Piotr formally
```

Comment:
When you first mention people, always link profiles, e.g., my LinkedIn and Marek's Github. Good links to profiles should be provided in people's profiles in the agenda.